lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-module/src/main/java/io/lighty/applications/rnc/module/config/SecurityConfig.java

```diff
@@ -14,22 +14,33 @@
 
 public class SecurityConfig {
     private final KeyStore keyStore;
+    private final KeyStore trustKeyStore;
     private final String password;
+    private final String trustPassword;
     private final SslContextFactory sslContextFactory;
+    private final boolean isNeedClientAuth;
 
-    public SecurityConfig(final KeyStore keyStore, final String password) {
+    public SecurityConfig(final KeyStore keyStore, final String password,
+                        final KeyStore trustKeyStore, final String trustPassword,
+                        final boolean isNeedClientAuth) {
```

Comment:
Please align these parameters, and if it makes sense, you can use 120 characters in one line.
```java
    public SecurityConfig(final KeyStore keyStore, final String password, final KeyStore trustKeyStore,                
                          final String trustPassword, final boolean isNeedClientAuth) {
```
Or, if you like, you can divide method parameters and code with one tab indentation.
```java
    public SecurityConfig(final KeyStore keyStore, final String password, final KeyStore trustKeyStore, 
            final String trustPassword, final boolean isNeedClientAuth) {
```